src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Validation.cs

```diff
@@ -720,5 +732,17 @@
             scalarCountAdjustment = tempScalarCountAdjustment;
             return pInputBuffer;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static uint AdvSimdMoveMask(Vector128<byte> value, Vector128<byte> mostSignficantBitMask)
+        {
+            Debug.Assert(AdvSimd.Arm64.IsSupported);
+
```

Comment:
>Vector128.Create((ushort)0x1001).AsByte(); [](start = 35, length = 42)

You still want to declare this outside the loop and pass it as parameter otherwise we will execute `Vector128.Create()` in every iteration.

Reply:
Good catch. Fixing.